src/components/ProjectCard/ProjectCard.styles.tsx

```diff
@@ -6,27 +6,36 @@ import { Icon } from "../Icon/Icon";
 
 export const Wrapper = styled.div`
   display: flex;
-  flex-flow: column;
-  max-width: 20rem;
+  flex-flow: row;
   width: 100%;
-  min-width: 16rem;
-  gap: 0.5rem;
+  height: 100%;
+  gap: 1rem;
+  justify-content: space-between;
+  max-width: 700px;
+
+  @media (max-width: 675px) {
+    flex-flow: column;
+  }
 `;
 
 export const Link = styled.a`
-  border-radius: 4px;
-  overflow: hidden;
-  box-shadow: 0px 0px 2px #616161;
+  text-decoration: none;
+  width: 100%;
+  height: auto;
+`;
+
+export const ThumbContainer = styled.div`
+  position: relative;
+  aspect-ratio: 16/9;
   width: 100%;
   height: auto;
-  margin: 0;
-  padding: 0;
 `;
 
 export const Thumb = styled(Image)`
```

Comment:
The scale for this component would be more graceful if it was just 1.01, it's just an indicator that something can be clicked